docs-cg.md

```diff
@@ -0,0 +1,67 @@
+  - Identify needs for Guides, How-Tos, and Tutorials and create them.
+- Work with specification authors to define best practices for web developer docs, code examples, and information architecture.
+- Maintain and keep documentation up-to-date using modern approaches like docs-as-code and develop tools to create feedback loops between standards and web developer documentation.
+- Collaborate with other horizontal groups such as WebDX, Security, Privacy, Accessibility, Internationalization, Sustainability and integrate their findings and best practices into docs.
+- Create a map of documentation sources for web developers.
+  - Identify reliable sources of information.
+  - Invite them to be part of the conversations in the CG.
+- Drive adoption (or deprecation!) of web platform technologies.
+  - Identify web platform features where sparse docs prevent adoption and understanding.
+  - Interpret developer surveys (such as the [MDN short survey on web Developer’s experience with Web Security](https://github.com/web-platform-dx/developer-research/blob/main/mdn-short-surveys/2023-05-15-security-dx/interpretation.md) and understand the role of documentation in identified shortcomings.
+- Coordinate horizontal efforts to get outdated docs updated by, e.g. helping reach out to the right docs owners.
+- Act as a hub for when docs-related efforts/changes/projects appear, so people can get these amplified/coordinated with the relevant stakeholders.
+
+## Key deliverables
+
+- Write and maintain docs in [mdn/content](https://github.com/mdn/content). See the [backlog](https://openwebdocs.github.io/web-docs-backlog/) for missing reference pages.
+- Identify best practices for web developer documentation.
+- Drive adoption and understanding of web technologies.
+- Act as a hub to coordinate documentation projects.
+
+## Out of scope
+
+The group may produce Community Group Reports within the scope of this charter but not Specifications.
+
+## Dependencies or liaisons
+
+- Groups
+  - Open Web Docs
+  - Web Developer Experience (WebDX) Community Group
+  - Webref & spec tooling maintainers
+  - SWAG Community Group
+  - TAG (especially related to “explainers”)
+  - OWASP docs
+  - [OSTIF](https://ostif.org/)
+  - Write The Docs community
+  - ARIA Practices Guide Taskforce (a subgroup of ARIA WG) who owns/maintains [APG](https://www.w3.org/WAI/ARIA/apg/).
+- Docs sites
+  - MDN Web Docs
+  - CanIUse
+  - web.dev
+  - learn.microsoft.com
+  - devdocs.io
+  - Freecodecamp
+  - w3schools
+  - Popular front-end sites/blogs which also document the web, for example: [css-tricks.com](https://css-tricks.com/almanac/pseudo-selectors/)
+  - Translated docs, for example: [Doka: web docs for Russian speaking developers](https://doka.guide/)
+
+## Audience
+
+Who is this CG for?
+
+- Technical Writers to talk amongst themselves.
+- Web platform docs sites to share best practices.
+- Specification authors and implementers to give and to get feedback on docs and specs.
```

Comment:
I suppose the intent is not to invite spec authors to come to the group and ask for general dev feedback on their spec, but more to get feedback on the spec from a documentation perspective.

```suggestion
- Specification authors and implementers to give feedback on docs and get feedback from technical writers and docs sites on specs.
```

Reply:
```suggestion
- Specification authors and implementers to give and receive feedback on documentation and specs.
```

See previous comment re: role naming. Otherwise, I think @tidoust's suggestion is great.